tests/method/tmultim.nim

```diff
@@ -10,7 +10,7 @@ collide: thing, unit |
 do nothing
 '''
   joinable: false
-  disabled: true
+    disabled: true
```

Comment:
Disabled iaround 2020-03 in a commit that references #13803, for CI sake?
		Working in my pc.